internals-js/src/buildSchema.ts

```diff
@@ -315,7 +317,9 @@ function buildNamedTypeInner(
       break;
   }
   buildAppliedDirectives(definitionNode, type, extension);
-  type.description = definitionNode.description?.value;
+  if (definitionNode.description) {
```

Comment:
same as above.

Reply:
That's the part I'm referring to in the "Note" of the commit message: it's not the same behaviour in that this code may run multiple times for the same `type`, so if we had a description the first time (say on the type definition), then we were erasing it if there was no description the 2nd time (say on the type extension). The change makes it so we don't erase it anymore.